packages/typespec-go/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 0.9.0 (UNRELEASED)
```

Comment:
This should be `0.8.3` since there's no breaking change. Please be sure to also update `package.json` with the matching version number.